dbtmetabase/__init__.py

```diff
@@ -1,59 +1,115 @@
 import logging
+import sys
+import os
 
-from .dbt import DbtReader
 from .metabase import MetabaseClient
+from .parsers.dbt_folder import DbtFolderReader
+from .parsers.dbt_manifest import DbtManifestReader
 
-__version__ = "0.7.0"
+from typing import Mapping, Iterable, List, Union
+
+__version__ = "0.8.0"
 
 
 def export(
-    dbt_path: str,
     mb_host: str,
     mb_user: str,
     mb_password: str,
     database: str,
-    schema: str,
-    mb_https=True,
-    sync=True,
-    sync_timeout=30,
```

Comment:
Is it intentional that the default value of 30 was removed for `sync_timeout`?

Reply:
The default of 30 still lives at the innermost function call so if sync timeout is not specified, default time is still unchanged. The difference now is explicitly passing in the arg `sync_timeout` will cause fail fast behavior whereas not passing it in attempts sync for default of 30 seconds and continues execution regardless of exact match. This gives users finer control without effecting how they interface with the cli.